src/mongocxx/private/collection.hpp

```diff
@@ -38,8 +38,7 @@ class collection::impl {
          const class client::impl* client, stdx::string_view name)
```

Comment:
Since `name` is no longer used here, let's remove it. I think the only place we call this constructor is collection.cpp line 83.